exporter/otlphttpexporter/otlp.go

```diff
@@ -120,7 +154,16 @@ func (e *baseExporter) export(ctx context.Context, url string, request []byte, p
 	if err != nil {
 		return consumererror.NewPermanent(err)
 	}
-	req.Header.Set("Content-Type", protobufContentType)
+
+	switch e.config.Encoding {
+	case EncodingJSON:
+		req.Header.Set("Content-Type", jsonContentType)
+	case EncodingProto:
+		req.Header.Set("Content-Type", protobufContentType)
+	default:
```

Comment:
Since we have explicit cases for the encodings we handle I think this should error instead of fallback to proto.

Reply:
Since there is an issue with adding the default value explicitly (breaking contrib) I cant change this, unless breaking contrib is acceptable (I can create a small fix in the `f5cloudexporter` once this change is merged)